src/homedelivery/export.ts

```diff
@@ -203,12 +232,12 @@ async function processSubs(
 	 * 'Body: stream' params field, it does not seem to work with the stream provided by csv-parser,
 	 * thus we had to convert the stream to string using get-stream package.
 	 */
-	const streamAsString = await getStream(stream);
+	const streamAsString = await getStream(stream as ReadStream);
```

Comment:
What is the type before `as`?

Reply:
I think it is `any`.

But I believe it should be a ReadStream.

It's worth noting that in this migration I didn't want to change any code behaviour, so it is entirely possible that there is a better, more modern way to handle those streams which I can look at in later PRs.

Reply:
Yes, that approach is sensible